inc/mlx90632.h

```diff
@@ -66,7 +66,7 @@
 #ifndef GENMASK
 #ifndef BITS_PER_LONG
 #warning "Using default BITS_PER_LONG value"
-#define BITS_PER_LONG 64 /**< Define how many bits per long your CPU has */
+#define BITS_PER_LONG 32 /**< Define how many bits per long your CPU has */
```

Comment:
Unrelated change, please leave the default value unchanged.

Reply:
The default value was 64, right?

Reply:
yes.